core/src/main/java/org/apache/carbondata/core/datastore/columnar/BlockIndexerStorageForNoInvertedIndexForShort.java

```diff
@@ -17,85 +17,17 @@
 
 package org.apache.carbondata.core.datastore.columnar;
 
-import java.util.ArrayList;
-import java.util.List;
-
-import org.apache.carbondata.core.constants.CarbonCommonConstants;
-import org.apache.carbondata.core.util.ByteUtil;
-
 /**
  * Below class will be used to for no inverted index
+ * Only try applying RLE on data
  */
 public class BlockIndexerStorageForNoInvertedIndexForShort extends BlockIndexerStorage<byte[][]> {
```

Comment:
Please change class name to a more understandable one

Reply:
done